lib/config/rules/functionality.js

```diff
@@ -72,7 +72,7 @@ module.exports = {
   // Require a `default` case in all `switch` statements.
   'switch-default': false,
   // Requires `===` and `!==` in place of `==` and `!=`.
-  'triple-equals': true,
+  'triple-equals': [true, 'allow-null-check'],
```

Comment:
This line is good.  I'm assuming the rest of the PR was accidentally submitted.

I'm guessing that you're accidentally creating branches from your current branch.  Always `git checkout master && git pull && git checkout -b my_new_branch`.